src/sh_of_action.mli

```diff
@@ -0,0 +1,5 @@
+(** Convert an action to a shell command suitable for [/bin/sh] *)
+
+open Stdune
+
+val pp : Action.For_shell.t -> unit Pp.t
```

Comment:
A bit of a bike shed, but I think the name `action_sh` or `action_to_sh` is better. Since we already have other action related modules that have the same prefix.